Code/src/main/java/nl/utwente/group10/ui/components/ConnectionLine.java

```diff
@@ -0,0 +1,65 @@
+package nl.utwente.group10.ui.components;
+
+import java.io.IOException;
+
+import nl.utwente.ewi.caes.tactilefx.control.TactilePane;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.shape.CubicCurve;
+
+
+/**
+ * This class represent a Connection-Line visual object in the UI.
+ * Each UI element that uses Connection-Line properties should extend
+ * from here.
+ * 
+ * For Lines based on Start and End anchors, see AnchoredConnectionLine
+ * For Lines that connect inputs and outputs of Blocks see Connection.
+ */
+public class ConnectionLine extends CubicCurve {
+	/** Control offset for this bezier of this line */
+	public static final double BEZIER_CONTROL_OFFSET_Y = 100f;
+
+	/** The fxmlLoader responsible for loading the fxml.*/
+	private FXMLLoader fxmlLoader;
+	
+	public ConnectionLine(){
+		try {
+			fxmlLoader = new FXMLLoader(getClass().getResource("/ui/ConnectionLine.fxml"));
+			fxmlLoader.setRoot(this);
+			fxmlLoader.setController(this);
+			fxmlLoader.load();
+		} catch (IOException e) {
+			// TODO Auto-generated catch block
+			e.printStackTrace();
```

Comment:
Hier kun je niet echt iets nuttigs doen denk ik, maar ik zou in ieder geval de TODO aanpassen naar iets van 'clean up loading fxml' in plaats van wat Eclipse er neer plempt.